src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -0,0 +1,1259 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "other_clinical_attribute"
+title: Other Clinical Attribute
+type: object
+namespace: https://gdc.cancer.gov
+category: clinical
+project: '*'
+program: '*'
+description: >-
+  Additional clinical attributes that are extraneous to other specific clinical entity
+  types within the GDC Data Dictionary.
+additionalProperties: false
+submittable: true
+downloadable: false
+previous_version_downloadable: false
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+      - name: cases
+        backref: other_clinical_attributes
+        label: describes
+        target_type: case
+        multiplicity: many_to_one
+        required: false
+      - name: follow_ups
+        backref: other_clinical_attributes
+        label: describes
+        target_type: follow_up
+        multiplicity: many_to_one
+        required: false
+
+required:
+  - submitter_id
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+properties:
+
+  $ref:
+    - "_definitions.yaml#/ubiquitous_properties"
+
+  aids_risk_factors:
+    $ref:
+      - "_terms.yaml#/aids_risk_factors/common"
+    enum:
+      - Candidiasis
+      - Coccidioidomycosis
+      - Cryptococcosis
+      - Cryptosporidiosis, Chronic Intestinal
+      - Cytomegalovirus
+      - Encephalopathy
+      - Herpes Simplex Virus
+      - Histoplasmosis
+      - Isosporiasis
+      - Mycobacterium avium Complex
+      - Mycobacterium tuberculosis
+      - Mycobacterium, NOS
+      - Nocardiosis
+      - Pneumocystis Pneumonia
+      - Pneumonia, NOS
+      - Progressive Multifocal Leukoencephalopathy
+      - Salmonella Septicemia
+      - Toxoplasmosis
+      - Wasting Syndrome
+    enumDef:
+      Pneumonia, NOS:
+        $ref:
+          - "_terms_enum.yaml#/pneumonia_nos/common"
+
+  bmi:
+    $ref:
+      - "_terms.yaml#/bmi/common"
+    type: number
+    minimum: 0
+
+  body_surface_area:
+    $ref:
+      - "_terms.yaml#/body_surface_area/common"
+    type: number
+    minimum: 0
+
+  cd4_count:
+    $ref:
+      - "_terms.yaml#/cd4_count/common"
+    type: number
+    minimum: 0
+
+  cdc_hiv_risk_factors:
+    $ref:
+      - "_terms.yaml#/cdc_hiv_risk_factors/common"
+    enum:
+      - Hemophiliac
+      - Heterosexual Contact
+      - Homosexual Contact
+      - Intravenous Drug User
+      - None
+      - Transfusion Recipient
+      - Unknown
+      - Not Reported
+    enumDef:
+      None:
```

Comment:
Aren't the other enum values required here?

Reply:
Yeah good question, we mainly add enum definitions if they have a standardized definition and associated reference id/code from a source like NCIt or caDSR; if these do not exist yet, a team called EVS (enterprise vocabulary services) will (hopefully) generate a new code with an associated definition for us to add in a later release. This is something we have been discussing internally since we do a have a backlog of mainly enums that do not have associated definitions/refs. 

So in this case we do not have, or have not had, enum defs generated yet or this is an upcoming action to add these in another new release.